### PR TITLE
Symlink aws-exports for Examples

### PR DESCRIPTION
**Pull request by slaymance**
*Issue #, if available:* [Asana](https://app.asana.com/0/1201410500481323/1201335765418250)

*Description of changes:* Symlink `aws-exports` for all examples to better reflect how a library user would import `aws-exports`.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,5 +6,6 @@
 node_modules
 aws-exports.js
 aws-exports.ts
+!examples/**/aws-exports.js
```
Comment (wlee221) [inline .gitignore:9]: Smart!

**Review by wlee221 — APPROVED** [review]
Looks good!

File: examples/angular/src/pages/ui/components/authenticator/custom-sign-up-fields/aws-exports.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import awsExports from '@environments/auth-with-email-and-custom-attributes/src/aws-exports';
+export default awsExports;
```
Comment (ericclemmons) [inline examples/angular/src/pages/ui/components/authenticator/custom-sign-up-fields/aws-exports.js:2]: I assume symlinking didn't work and instead these had to be explicit exports?

Also, I think this works too:

```js
export * from '@environments/auth-with-email-and-custom-attributes/src/aws-exports';
```

Or
```
export default from '@environments/auth-with-email-and-custom-attributes/src/aws-exports';
```

**Review by ericclemmons — COMMENTED** [review]
Definitely an improvement! Thanks @slaymance!

**Pull request merged**